src/OpenFOAM/meshes/primitiveMesh/PrimitivePatch/PrimitivePatch.H

```diff
@@ -117,7 +117,7 @@ private:
 
         //- Reference to global list of points
         PointField points_;
-
+        mutable gpuField<PointType>* gpuPointsPtr_;
```

Comment:
It checks if pointer is NULL, if it is not NULL it performs delete and then sets it to NULL. It is just a shorthand for a common operation.